portfolio/src/main/webapp/index.html

```diff
@@ -12,12 +12,20 @@ <h1>Salvador's Portfolio</h1>
       <p>This is Salvador's portfolio.</p>
       <p>I'm currently a STEP intern at Google from The University of Texas at El Paso.</p>
       <img src = "/Images/Me.jpg" class = "myPortrait" />
+
       <p>Click here to get a random fact about me:</p>
       <button onclick="addRandomFact()">Push me!!</button>
       <div id="random-facts-container"></div>
-      <p>Display messages:</p>
-      <button onclick="displayMessages()">Click me!</button>
-      <ol type="I" id="message-container"></ol>
+      <p>Display previous comments:</p>
+      <button onclick="displayComments()">Click me!</button>
+      <ol type="I" id="comments-container"></ol>
+
+      <form action="/data" method="POST" id="commentForm">
+        <p>Comment something...</p>
+        <textarea name="text-comment">Type here...</textarea>
+      </form>
+
+      <button type="submit" form="commentForm">Submit</button>
```

Comment:
This is important. The submit button finds the enclosing `<form>` tags. Right now, it's outside of that, so it will never submit and nothing will submit the comment in the textarea.

Reply:
I put a form="commentForm" inside the button tag, I thought that helped because it works